### ISSUE TITLE
Add stainless-dotty script to release zip file and package a default .stainless-conf

### ISSUE THREAD

**Issue by vkuncak** [epfl-lara/stainless#603]
* package into release file not only stainless-scalac but also stainless-dotty (keep the names)
  * add .stainless-conf into release to educate users about its existence

**Comment by romac** [comment]
Wouldn't it be better to make the config file visible by default by naming it `stainless.conf` instead? Both names are supported, so there wouldn't be any changes necessary in Stainless.

**Comment by vkuncak** [comment]
Agreed, making it visible is better; I missed that the other form is supported.
